azure/functions/servicebus.py

```diff
@@ -1,3 +1,4 @@
+import json
```

Comment:
Curious - don't we add any license headers for this library? We can do that in another PR, just asking.

Reply:
This is a good catch, I can do that in a separate PR.